apptail/cmd/apptail/main.go

```diff
@@ -34,9 +56,13 @@ func main() {
 
 	go docker.DockerListener.Listen()
 
+	// clean up the cache after restart
+	docker.DockerListener.TrackerCleanUp(tracker)
```

Comment:
Should this be before the `Listen()` call instead?  I suppose it doesn't actually matter... hmm.